{"id": "CVE-2021-3737", "product": "python", "version": "n/a", "vulnerability": ["CWE-835 - Loop with Unreachable Exit Condition ('Infinite Loop'), CWE-400 - Uncontrolled Resource Consumption"], "description": "A flaw was found in python. An improperly handled HTTP response in the HTTP client code of python may allow a remote attacker, who controls the HTTP server, to make the client script enter an infinite loop, consuming CPU time. The highest threat from this vulnerability is to system availability.", "poc": {"reference": ["https://www.oracle.com/security-alerts/cpujul2022.html"], "github": ["https://github.com/ARPSyndicate/cvemon", "https://github.com/GitHubForSnap/matrix-commander-gael", "https://github.com/NathanielAPawluk/sec-buddy"]}}